CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ The types of changes are:
 
 ## [Unreleased](https://github.com/ethyca/fides/compare/2.40.0...main)
 
+### Changed
+ - Removed support for Python 3.8
+
```

Comment:
Drive-by comment: this doesn't remove support for Python 3.8, it just removes the testing of it 

Slack thread: https://ethyca.slack.com/archives/C04S7K0T2V7/p1720035732012629?thread_ts=1720029377.583919&cid=C04S7K0T2V7

Fideslang specifically restricts Python versions I believe, so my guess is that we'd want Fideslang to go out as well. I am already doing this for Pydantic V2 -

Reply:
https://github.com/ethyca/fideslang